hack/tool/backup_restore/README.md

```diff
@@ -0,0 +1,197 @@
+# Backup & Restore
+
+This go module is a command line tool to back up and restore the configuration and the Terraform state.
+
+It has two subcommands `backup` and `restore`.
+
+## `restore`
+
+The main usage of the `restore` subcommand is to import an "outside" Terraform instance (maybe created by the terraform command line or managed by another terraform-controller before) to the terraform-controller in the target kubernetes without recreating the cloud resources.
+
+To use the `restore` subcommand, you should:
+
+1. Prepare the `configuration.yaml` file and the `state.json` file whose content is the state json of the Terraform instance which would be imported.
+
```

Comment:
This is a typical scenario, great! Here is another one, please cover it in this PR or in a later one https://github.com/kubevela/terraform-controller/issues/310#issuecomment-1133548163.

Reply:
Is this should be covered by `backup` command?

Reply:
I can add the `backup` command in the next PR.